pkg/operator/ceph/cluster/watcher.go

```diff
@@ -262,11 +262,22 @@ func listRBDPV(listPVs *corev1.PersistentVolumeList, cluster *cephv1.CephCluster
 	var listRbdPV []corev1.PersistentVolume
 
 	for _, pv := range listPVs.Items {
-		// Ignore PVs that support multinode access (RWX, ROX), since they can be mounted on multiple nodes.
-		if pvSupportsMultiNodeAccess(pv.Spec.AccessModes) {
+		// Skip if pv is not provisioned by CSI
+		if pv.Spec.CSI == nil {
+			logger.Infof("pv %q is not provisioned by CSI", pv.Name)
```

Comment:
How about making this a debug log instead of info?

Reply:
done